package.json

```diff
@@ -4,7 +4,6 @@
   "description": "",
   "main": "dist/index.js",
   "browser": "dist/index.js",
-  "dependencies": {},
```

Comment:
Try to avoid useless changes

Reply:
It's not useless, please have a look at https://github.com/aeternity/superhero-utils/blob/2ab1d46ff3dd03457e89a67fba3d6374fd729984/package.json#L7